frontend/app/src/composables/history/events/mapping.ts

```diff
@@ -198,7 +194,7 @@ export const useHistoryEventMappings = createSharedComposable(() => {
   const { scrambleData, scrambleHex } = useScramble();
 
   const getEventCounterpartyData = (
-    event: MaybeRef<EvmHistoryEvent | EthDepositEvent>
+    event: MaybeRef<{ counterparty?: string | null; address?: string | null }>
```

Comment:
When is the type null and when is it undefined?

Reply:
Updated, so the counterparty shouldn't be missing (but `string | null`). For the address it's string | null if it's from event. But for the usage in the accounting rule, I don't send it.

Reply:
Updated, so the counterparty shouldn't be missing (but `string | null`). For the address it's string | null if it's from event. But for the usage in the accounting rule, I don't send it.